02-wrangling.Rmd

```diff
@@ -23,14 +23,13 @@ By the end of the chapter, students will be able to:
     - `group_by`
     - `map`
 
-## Vectors and Data frames
+## Data frames and Vectors
 
-At this point, we know how to load data into R from various file formats. Once loaded into R, most of the tools we have learned about for reading data into R represent the data as a data frame. So now we will spend some time learning more about data frames in R so that we have a better understanding 
-of how we can use and manipulate these objects.
+At this point, we know how to load data into R from various file formats. Once loaded into R, most of the tools we have learned about for reading data into R represent the data as a data frame. So now we will spend some time learning more about data frames in R so that we have a better understanding of how we can use and manipulate these objects.
 
 ### What is a data frame?
 
-Let's first start by defining what a data frame is exactly. From a data perspective, it is a rectangle where the rows are the observations:
+Let's first start by defining what a **data frame** is exactly. From a data perspective, it is a rectangle where the rows are the observations:
```

Comment:
I think we want to prefix observations and variables with "statistical". The reason for this, is that if we leave it as variables, then computer science folks will think we mean variables in the programming sense. This is not what we mean and I ran into several students who made this incorrect assumption this year.


We might then want to add a sentence defining what a statistical observation is, and what a statistical variable is.

Reply:
I said, "in statistics, we define..." instead of using the terms "statistical observation" -- I think it gets the same point across, but uses definitions more consistent with other resources